questions.md

```diff
@@ -0,0 +1,5 @@
+# Some questions about pull requests:
+
+*  what happens with multiple branches?
```

Comment:
When it comes to PRs you can only compare two branches at a time. If you have other branches you can compare those branches together and merge them if they are related in some way. Generally speaking I would recommend avoiding branching from branches. This gets messy over time. It is better to branch off your master branch.